images/subscription-notification.svg

```diff
@@ -0,0 +1 @@
 6.35 0 0 1-4.93 1.73c-3-.27-11.4-10.28-13.24-12.11-2.58-2.6-2.72-14.24-1.36-20.03z" fill="#e5bc93"/><path d="M314.9 129.33a32.53 32.53 0 0 0 12.1 4.32c6.27.65 13-.43 18.17 6.06s1 14.13-5 18.38c-4.54 3.25-10.38-13.41-26.82-13a17.06 17.06 0 0 1 5.41-8.44 12.87 12.87 0 0 1-3.86-7.32z" fill="#fff"/><circle cx="371.36" cy="93.85" r="5.19"/><circle cx="322.04" cy="80.87" r="5.19"/><path d="M317 382l.57-1.45a113.39 113.39 0 0 1 14.88-27.33.33.33 0 1 0-.53-.38 113.35 113.35 0 0 0-14.38 26c-.28-10 .29-23.87 4.24-35.78l.39-1.16-.92.82c-11.68 10.57-19.57 20.62-23.46 29.93 1.1-9.94 4.44-29.23 8.24-38.7l.55-1.36-1.07 1c-6.95 6.51-16.67 25.63-22.47 37.05l-.15.28c.73-9.83 4-31.47 10.44-42.75l.88-1.54-1.37 1.13c-7.25 6-16.35 16.7-20.89 29.14 0-8.06.18-20.71 4.32-30.68l.51-1.22-1 .85c-8.83 7.4-15.33 18-20.56 26.7 0-9.91 1.83-23.18 6.07-31.55l.43-.84-1 .5c-7.32 3.47-13.11 6.22-17.57 13.1a.33.33 0 0 0 .1.45.32.32 0 0 0 .45-.09c4.23-6.53 9.73-9.28 16.65-12.57-4.15 8.83-5.87 22.38-5.72 32.16v1.13l.79-1.32c5.08-8.43 11.35-18.85 19.82-26.32-3.89 10.33-3.89 23-3.89 30.75v2.95l.64-2c4-12.34 12.71-23.13 20-29.56-6.28 12.4-9.36 34.5-9.81 43.08l-.08 1.52 1.56-3c5.49-10.8 14.49-28.51 21.26-35.78-3.92 10.83-7.24 31.32-8 39.82l-.22 2.53.85-2.39c3.39-9.59 11.21-20 23.25-31.09-3.84 12.36-4.22 26.54-3.85 36.4zM100.82 170.31l.25-.46c3.9-7.24 4.09-23.59-.71-31.89 10.61 6.58 13 14.47 16.3 25.24.29 1 .6 2 .92 3l.39 1.27.31-1.29c.19-.81.39-1.61.58-2.42 2.48-10.15 5-20.63.36-31.77 9 9.18 13.25 17.91 15.43 31.52l.17 1.1.51-1c5-9.82 2.81-27.32.1-37 12.55 12.32 17.78 30 19.53 35.93l.39 1.31.31-1.33c3.28-13.92 5.69-27.44-.8-39.82l-.64.34c6.16 11.76 4.13 24.67 1.05 38-2.15-7.14-7.78-24.19-20.32-35.88l-1-.92.4 1.3c2.75 8.94 5.27 26.46.82 36.73-2.37-13.8-7-22.61-16.84-32.17l-1.49-1.45.91 1.87c5.65 11.59 3 22.52.41 33.1-.09.38-.19.76-.28 1.14-.18-.59-.36-1.17-.53-1.74-3.46-11.37-6-19.58-18-26.45l-1.49-.85 1 1.38c5.42 7.4 5.55 24.24 1.76 32a42.36 42.36 0 0 1-5.07-8.77c-2.3-4.81-4.69-9.78-9.54-13.48l-.44.58c4.71 3.59 7.06 8.49 9.33 13.22 1.63 3.41 3.17 6.64 5.52 9.34z"/><path d="M64.8 144.46c8.33 7.06 22.63 27.19 22.25 41.8l-11.13 5.39c-1.16-13.54-16.13-32.83-22.92-39.1z" fill="#754c24"/><circle cx="74.19" cy="167.78" r="13.63" fill="#f8f8fa"/><path d="M73.29 156.92a10.9 10.9 0 1 1-10 11.77 10.9 10.9 0 0 1 10-11.77m-.23-2.72a13.63 13.63 0 1 0 14.71 12.46 13.63 13.63 0 0 0-14.71-12.46z" fill="#7d7663"/><path d="M176.11 313.82c-11.13-16.51-6.1-15.21-11.86-22.33 2.76 5.42 2.91 8.51 1.93 8.83-1.44.24-2.74 1-4.93 3.32 3.32-1.71 4.61-1.4 7.05-.41z" fill="#a06c60"/><path fill="#fff" d="M75.61 168.35l-6.63-7.08"/><path d="M75.69 169.37a1 1 0 0 0 .61-.27 1 1 0 0 0 0-1.45l-6.63-7.08a1 1 0 0 0-1.44 1.43l6.63 7.09a1 1 0 0 0 .83.28z"/><path d="M72.75 175.2a1 1 0 0 0 .82-.56l3-5.83a1 1 0 0 0-1.83-.92l-2.94 5.83a1 1 0 0 0 .45 1.37 1.07 1.07 0 0 0 .5.11z"/><path d="M43.18 235.32c4.27-2.78 8.06-5.55 9.7-8.29 0 0 4.64.88 7.74-1.62.42 1.88-5.62 3.68-7.4 3.34-1.33 2.34-5.57 4.46-10.04 6.57z" fill="#a06c60"/></svg>
```

Comment:
I see the banner has been removed. This SVG file should be removed as well.
If it needs to stay for some reason, it should get the same a11y treatment we apply to all the SVGs:
- remove the title tag
- add `role="img" aria-hidden-"true" focusable="false"`